src/soc/ibm/power9/istep_13_2.c

```diff
@@ -0,0 +1,158 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <cpu/power/istep_13.h>
+#include <console/console.h>
+#include <cpu/power/scom.h>
+#include <cpu/power/scom_registers.h>
+#include <timer.h>
+
+/*
+ * 13.2 mem_pll_reset: Reset PLL for MCAs in async
+ *
+ * a) p9_mem_pll_reset.C (proc chip)
+ *    - This step is a no-op on cumulus as the centaur is already has its PLLs
+ *      setup in step 11
+ *    - This step is a no-op if memory is running in synchronous mode since the
+ *      MCAs are using the nest PLL, HWP detect and exits
+ *    - If in async mode then this HWP will put the PLL into bypass, reset mode
+ *    - Disable listen_to_sync for MEM chiplet, whenever MEM is not in sync to
+ *      NEST
+ */
+void istep_13_2(dimm_attr dimms[CONFIG_DIMM_MAX])
+{
+	printk(BIOS_EMERG, "starting istep 13.2\n");
+	int d;
```

Comment:
Please change the variable name to something more meaningful.

Reply:
Changed.